### PR TITLE
Add support for deleting files from cloudstorage

### PR DESCRIPTION
**Pull request by mattss**

### CODE REVIEW

**Comment by wengole** [comment]
@mattss Cool. :+1:

File: src/netsight/cloudstorage/adapters.py [hunk 1]
```diff
@@ -97,10 +97,31 @@ def _getFields(self, ignore_empty=True):
         return result
 
     def _get_s3_connection(self):
+        """Set up an S3 connection using the registry settings"""
         aws_key = get_value_from_registry('aws_access_key')
         aws_secret_key = get_value_from_registry('aws_secret_access_key')
         return S3Connection(aws_key, aws_secret_key)
 
+    def _get_bucket(self):
```
Comment (adamcheasley) [inline src/netsight/cloudstorage/adapters.py]: There's duplicate code here. Suggest factoring out.

**Comment by adamcheasley** [comment]
Looks good other than the duplication.

**Comment by wengole** [comment]
~~Looks good other than the duplication.~~

**Comment by mattss** [comment]
:smiley_cat:

**Comment by mattss** [comment]
@adamcheasley Duplication removed.

**Pull request merged**